pkg/servicetest/servicetest_test.go

```diff
@@ -136,3 +153,46 @@ func TestServices(t *testing.T) {
 		}
 	}
 }
+
+// checkHTTP will attempt to communicate with a service over HTTP
+func checkHTTP(t *testing.T, container docker.APIContainers, targetPort int64, path string) {
+	t.Log("Checking HTTP communication")
+	assert := asrt.New(t)
+
+	for _, port := range container.Ports {
+		if port.PrivatePort == targetPort && port.PublicPort != 0 {
+			address := fmt.Sprintf("http://127.0.0.1:%d%s", port.PublicPort, path)
```

Comment:
This is a decent test, but I think it may be leaving out the important fact that we have to be able to access the port from *within* the web container. I imagine this approach is what we were doing before, but we should consider using app.Exec() to take an action within the web container. nc (netcat) is installed in there, it would do fine I think.